developer/requirements.md

```diff
@@ -2,22 +2,22 @@
 
 ## macOS
 
-- We suggest installation of [Xcode](https://developer.apple.com/xcode/downloads/)
+- To install the required build tools, Mac users must install [Xcode](https://developer.apple.com/xcode/downloads/)
```

Comment:
I guess xcode must be required now, see: https://github.com/nodejs/node-gyp#installation